From 54a1dd57e9f610addbe8b48ad3bce36b71fdeff6 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Sun, 11 Jan 2015 14:28:08 -0800
Subject: [PATCH 1/9] libunwind: Add missing include of libunwind_ext.h to
 CompactUnwinder.hpp, DwarfInstructions.hpp, and UnwindCursor.hpp

http://www.llvm.org/bugs/show_bug.cgi?id=22202

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit e80eb74721aab2cb6cea3bd24f3e6519741d211b)
---
 src/CompactUnwinder.hpp   | 1 +
 src/DwarfInstructions.hpp | 2 +-
 src/UnwindCursor.hpp      | 1 +
 3 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/CompactUnwinder.hpp b/src/CompactUnwinder.hpp
index cd9ce3e..44de2c6 100644
--- a/src/CompactUnwinder.hpp
+++ b/src/CompactUnwinder.hpp
@@ -21,6 +21,7 @@
 
 #include "AddressSpace.hpp"
 #include "Registers.hpp"
+#include "libunwind_ext.h"
 
 #define EXTRACT_BITS(value, mask)                                              \
   ((value >> __builtin_ctz(mask)) & (((1 << __builtin_popcount(mask))) - 1))
diff --git a/src/DwarfInstructions.hpp b/src/DwarfInstructions.hpp
index dfe45fb..c2d293f 100644
--- a/src/DwarfInstructions.hpp
+++ b/src/DwarfInstructions.hpp
@@ -22,7 +22,7 @@
 #include "Registers.hpp"
 #include "DwarfParser.hpp"
 #include "config.h"
-
+#include "libunwind_ext.h"
 
 namespace libunwind {
 
diff --git a/src/UnwindCursor.hpp b/src/UnwindCursor.hpp
index 59924f0..ba04a4d 100644
--- a/src/UnwindCursor.hpp
+++ b/src/UnwindCursor.hpp
@@ -31,6 +31,7 @@
 #include "DwarfInstructions.hpp"
 #include "EHHeaderParser.hpp"
 #include "libunwind.h"
+#include "libunwind_ext.h"
 #include "Registers.hpp"
 #include "Unwind-EHABI.h"
 
-- 
2.6.2

